tests/test_wrappers.py

```diff
@@ -1199,3 +1200,58 @@ def test_pprint(self):
             epochs=EPOCHS,
         )
         print(clf)
+
+
+class TestPartialFit:
+    @pytest.mark.parametrize(
+        "config", ["MLPRegressor", "CNNClassifier", "CNNClassifierF"],
+    )
+    def test_partial_fit(self, config):
+        loader, model, build_fn, _ = CONFIG[config]
+        clf = model(build_fn, epochs=1)
+        data = loader()
+
+        X, y = data.data[:100], data.target[:100]
+        clf.partial_fit(X, y)
+
+        # history_ records the history from this partial_fit call
+        # Make sure processes one epoch (zero based indexing)
+        hist = copy(clf.history_.history)
+        assert clf.history_.epoch == [0]
+        assert len(hist["loss"]) == 1
+
+        # Make sure new model not created
+        model = clf.model_
+        clf.partial_fit(X, y)
+        assert clf.model_ is model, "Model memory address should remain constant"
+
+    @pytest.mark.parametrize(
+        "config", ["MLPRegressor", "CNNClassifier", "CNNClassifierF"],
+    )
+    def test_pf_pickle_pf(self, config):
+        loader, model, build_fn, _ = CONFIG[config]
+        clf = model(build_fn, epochs=1)
+        data = loader()
+
+        X, y = data.data[:100], data.target[:100]
+        clf.partial_fit(X, y)
+
+        clf2 = pickle.loads(pickle.dumps(clf))
+        clf2.partial_fit(X, y)
+        assert len(clf.history_["loss"]) == 1
+        assert len(clf2.history_["loss"]) == 2
+        assert np.allclose(clf.history_["loss"][0], clf2.history_["loss"][0])
+        assert clf2.history_["loss"][1] <= clf2.history_["loss"][0]
```

Comment:
It does not seem like the loss reliably decreases. Do you feel strongly that this is a good test, or can we do without/come up with an alternative @stsievert?

Reply:
I've implemented a better test. I'm interested in making sure the optimizer and weights are hooked together. Now, I test that the weights have changed.

Reply:
I saw, left a comment on there. Thank you!